docs/installation/bootstrap.rst

```diff
@@ -47,21 +50,37 @@ Configuration
       archives:
         - <path-to-metalk8s-iso>
 
-The ``archives`` field is a list of absolute paths to MetalK8s ISO files. When
-the bootstrap script is executed, those ISOs are automatically mounted and the
-system is configured to re-mount them automatically after a reboot.
+
+The ``networks`` field specifies a range of IP addresses for pod networks and
+services in the Controlplane and Workloadplane written in CIDR notation.
+Note that after the Bootstrap node is installed, these values cannot be changed
+automatically.
+In production clusters, we advise users to choose a
+subnet range of atleast `/16` in order to have sufficient IP addresses for Pods
+and services keeping in mind that users infrastructure will grow in the near
+future. For example;
+
+   .. code-block:: shell
+
+      networks:
+        controlPlane: 10.200.0.0/16
+        workloadPlane: 10.200.0.0/16
 
 The ``proxies`` field can be omitted if there is no proxy to configure.
 The 2 entries ``http`` and ``https`` are used to configure the containerd
 daemon proxy to fetch extra container images from outstide the MetalK8s
 cluster.
 The ``no_proxy`` entry specifies IPs that should be excluded from proxying,
-it must a list of hosts, IP addresses or IP ranges in CIDR format.
+it must be a list of hosts, IP addresses or IP ranges in CIDR format.
+For example;
 
-.. todo::
+   .. code-block:: shell
 
-   - Explain the role of this config file and its values
+      NO_PROXY="localhost,127.0.0.1,10.10.0.0/16,192.168.0.0/16"
```

Comment:
This seems confusing: not in YAML-format, and not using the same key name (caps vs. lower). Also, the YAML value must be a list (type), not a comma-separate string.